data_rentgen/server/api/v1/router/auth.py

```diff
@@ -51,3 +64,15 @@ async def auth_callback(
     request.session["refresh_token"] = code_grant["refresh_token"]
 
     return {}
+
+
+@router.get("/logout")
+async def logout(
+    request: Request,
+    current_user: Annotated[User, Depends(get_user())],
+    auth_provider: Annotated[KeycloakAuthProvider, Depends(Stub(AuthProvider))],
+):
+    refresh_token = request.session.get("refresh_token", None)
```

Comment:
```suggestion
    refresh_token = request.session["refresh_token"]
```
Actually, as this method uses `get_user()`, endpoint will return 401 if user has no session cookie

Reply:
This is doesn't work for DummyAuthProvider.